wrappers/ortools.rb

```diff
@@ -50,7 +50,6 @@ def solver_constraints
         :assert_only_force_centroids_if_kmeans_method,
         :assert_no_scheduling_if_evaluation,
         :assert_route_if_evaluation,
-        :assert_no_shipments_if_evaluation,
```

Comment:
This should not be removed but assert should be edited with check on vrp.relations

Reply:
I see no reason to keep the assert in the first place. So I didn't port it to the more generic way